DI/ValidatorExtension.php

```diff
@@ -29,9 +29,14 @@ public function loadConfiguration()
 		if (!$builder->parameters['debugMode']) {
 			return;
 		}
-
+		
+		$presentersToDisableValidationFor = array();
+		if(isset($this->config['presentersToDisableValidationFor']) && is_array($this->config['presentersToDisableValidationFor'])) {
+			$presentersToDisableValidationFor = $this->config['presentersToDisableValidationFor'];
+		}
+		
 		$builder->addDefinition($this->prefix('panel'))
-			->setClass('Kdyby\Extension\Diagnostics\HtmlValidator\ValidatorPanel')
+			->setClass('Kdyby\Extension\Diagnostics\HtmlValidator\ValidatorPanel', Array($presentersToDisableValidationFor) )
```

Comment:
one last thing, we can do `$this->config['presentersToDisableValidationFor'] ?? []` and remove the whole block above

Meanwhile, I'd increase min PHP v to something like 7.1, would it work for you?

Reply:
This is little bit problem for me, because I use this extension on project where is PHP 5.6
Can you make this changes in another version in the future?

Reply:
I was afraid so. In that case lets change it to:

```php
$presentersToDisableValidationFor = isset($this->config['presentersToDisableValidationFor']) ? $this->config['presentersToDisableValidationFor'] : [];
```

What do you think?

Reply:
I made this change.